src/VisualStudio/Core/Def/Telemetry/VisualStudioWorkspaceTelemetryService.ProjectTelemetry.cs

```diff
@@ -4,27 +4,19 @@
 
 using System;
 using System.Collections.Immutable;
-using System.Composition;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
-using Microsoft.CodeAnalysis.ErrorReporting;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.ProjectTelemetry;
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.Internal.VisualStudio.Shell;
-using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 using Roslyn.Utilities;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectTelemetry
+namespace Microsoft.VisualStudio.LanguageServices.Telemetry
 {
-    [ExportEventListener(WellKnownEventListeners.Workspace, WorkspaceKind.Host), Shared]
-    internal class VisualStudioProjectTelemetryService
-        : ForegroundThreadAffinitizedObject, IProjectTelemetryListener, IEventListener<object>, IDisposable
+    internal partial class VisualStudioWorkspaceTelemetryService : IProjectTelemetryListener, IDisposable
```

Comment:
unified this with VisualStudioWorkspaceTelemetryService since to me it allowed me to easily chain with TelemetrySessionInitializedAsync (and logically made sense to me)